lib/web_ui/lib/src/engine/compositor/canvaskit_api.dart

```diff
@@ -1641,3 +1687,40 @@ external Object? get _finalizationRegistryConstructor;
 
 /// Whether the current browser supports `FinalizationRegistry`.
 bool browserSupportsFinalizationRegistry = _finalizationRegistryConstructor != null;
+
+@JS()
+class SkData {
+  external int size();
+  external bool isEmpty();
+  external Uint8List bytes();
+}
+
+@JS()
+@anonymous
+class SkImageInfo {
+  external SkAlphaType get AlphaType;
+  external int get BitsPerPixel;
+  external int get BytesPerPixel;
+  external int get BytesSize;
+  external int get BytesSize64;
+  external SkColorSpace get ColorSpace;
+  external SkColorType get ColorType;
+  external int get Height;
+  external bool get IsEmpty;
+  external bool get IsOpaque;
+  external SkRect get Rect;
+  external int get RowBytes;
+  external int get RowBytes64;
+  external int get Width;
+  external factory SkImageInfo({
```

Comment:
nit: Flutter style prefers putting factories and constructors above everything else.